samples/dnn/object_detection.py

```diff
@@ -163,9 +160,23 @@ def drawPred(classId, conf, left, top, right, bottom):
         print('Unknown output layer type: ' + lastLayer.type)
         exit()
 
-    indices = cv.dnn.NMSBoxes(boxes, confidences, confThreshold, nmsThreshold)
+    indices = []
+    if lastLayer.type == 'Region' and len(outNames) > 1:
+        classIds = np.array(classIds)
+        boxes = np.array(boxes)
+        confidences = np.array(confidences)
```

Comment:
Is that necessary?

Reply:
Yes, because `class_indices` may be a list. Onle `np.array` support a list of indices.

Reply:
But what about boxes and confidences?

Reply:
We need np.array(boxes) to get boxes[list_indices]